CHANGELOG.md

```diff
@@ -40,7 +43,8 @@
 * Adjusted `.travis.yml` checks to allow for `patch` branches to be tested
 * Add Python 3.7 dependency to the `environment.yml` file
 * Remove awsbatch profile cf [nf-core/configs#71](https://github.com/nf-core/configs/pull/71)
-* Make `scrape_software_versions.py` compatible with Python3 to enable miniconda3 in [base image PR](https://github.com/nf-core/tools/pull/462)
+* Make `scrape_software_versions.py` compatible with Python3 to enable miniconda3 in    [base image PR](https://github.com/nf-core/tools/pull/462)
+* Add `docker pull` command to GitHub actions CI workflow
```

Comment:
Changelog is getting a big strange for this release and could maybe do with an overhaul (eg. I think GitHub actions is new in this release? So doesn't really make sense to say that new stuff was added to GitHub Actions).

But not critical and can be handled for the release.

Reply:
I went over a bit the changelog now